### ISSUE TITLE
Would you mind release `redis-actionpack` to rubygems.org

### ISSUE THREAD

**Issue by larryzhao** [redis-store/redis-actionpack#19]
Would you mind release `redis-actionpack` as a minor release to rubygems.org so that we could use `redis-store 1.4.0`?

Thanks and Best Regards.

**Comment by tubbo** [comment]
You don't need a new version of redis-actionpack to take advantage of redis-store 1.4.0, we just don't require you to install redis-store 1.4.0 in order to use redis-actionpack. You can add the following to Gemfile to install redis-store 1.4.0 on your application:

```ruby
gem 'redis-store', '~> 1.4.0'
```

**Comment by larryzhao** [comment]
@tubbo I think you misunderstood me. The current version released to `rubygems.org` has the dependency specified as `'redis-store', '>= 1.1.0', '< 1.4.0'`. So if i install `redis-store 1.4.0`, I get:

```
Bundler could not find compatible versions for gem "redis-store":
  In Gemfile:
    redis-store (= 1.4.0)

    redis-rails (= 5.0.2) was resolved to 5.0.2, which depends on
      redis-activesupport (< 6, >= 5.0) was resolved to 5.0.3, which depends on
        redis-store (~> 1.3.0)
```

Since you've already updated the gemspec on master branch for months. Would you mind release a minor version to `rubygems.org`?

Thanks and Best Regards.

**Comment by tubbo** [comment]
@larryzhao I definitely misunderstood you...totally forgot to release what I did to RubyGems. Just released redis-actionpack 5.0.2.

**Comment by larryzhao** [comment]
@tubbo Thank you very much. You work helped us a lot.
